src/main/java/com/rarchives/ripme/ripper/rippers/ChanRipper.java

```diff
@@ -19,7 +19,7 @@
 
 public class ChanRipper extends AbstractHTMLRipper {
     public static List<ChanSite> explicit_domains = Arrays.asList(
-        new ChanSite(Arrays.asList("boards.4chan.org"),   Arrays.asList("4cdn.org", "is.4chan.org")),
+        new ChanSite(Arrays.asList("boards.4chan.org"),   Arrays.asList("4cdn.org", "is.4chan.org", "is2.4chan.org")),
```

Comment:
@rautamiekka You need to double \ in java so the regex would be is(|[0-9]+)\\.4chan\\.org